cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -194,3 +195,17 @@ This section comprises a glossary of all the keywords -- grouped by category and
 |Name           | Description
 | <<query-syntax-case, CASE Expression>>  | A generic conditional expression, similar to if/else statements available in other languages.
 |===
+
+
+[[glossary-preparser-options]]
+== Preparser options XX Docs Team: please harmonize
+
+[options="header"]
+|===
+|Name           | Type | Description
+| <<cypher-preparser-version, CYPHER $version query>>  | Version | This will force `'query'` to use Neo4j Cypher `$version`. The default is `3.2`.
+| <<cypher-preparser-planner, CYPHER planner=rule query>> | Planner |  This will force `'query'` to use the rule planner. As the rule planner has been removed in 3.2, doing this will cause `'query'` to fall back to using Cypher 3.1.
+| <<cypher-preparser-planner, CYPHER planner=cost query>> | Planner | Neo4j {neo4j-version} uses the cost planner for all queries.
+| <<cypher-preparser-runtime, CYPHER runtime=interpreted query>> | Runtime | This will force `'query'` to use the interpreted runtime. This is the only option in Neo4j Community Edition.
+| <<cypher-preparser-runtime, CYPHER runtime=compiled query>> | Runtime | This will force `'query'` to use the compiled runtime, if it is able to do so (otherwise `'query'` falls back to using the interpreted runtime). This is only available in Neo4j Enterprise Edition.
```

Comment:
I wonder if the word 'force' is compatible with the fact that it also 'falls back'?

Reply:
👍